docs/guides/scalar.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Custom Scalars
+---
+
+The GraphQL specification includes default scalar types Int, Float, String, Boolean, and ID. Although these scalars cover the majority of use cases, some applications need to support other atomic data types (such as Date) or add validation to an existing type.
+Tailcall supports a set of predefined custom scalars to enhance the functionality and validation of your GraphQL APIs
+
+Currently supported scalars:
+
+- `Email`
+- `PhoneNumber`
+- `Date`
+- `Url`
```

Comment:
convert this to a table with the Scalar a description of the specification that's followed and possible links for people to learn more about them.

Reply:
I think the table should also contain, the default scalars that are defined in the GraphQL spec.